Dockerfile

```diff
@@ -127,3 +128,14 @@ RUN chmod +x /tools/obodash && \
 
 # Install KGCL ROBOT plugin
 RUN wget -nv -O /tools/robot-plugins/kgcl.jar https://github.com/gouttegd/kgcl-java/releases/download/kgcl-java-$KGCL_JAVA_VERSION/kgcl-robot-plugin-$KGCL_JAVA_VERSION.jar
+
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+        wget -nv https://github.com/mikefarah/yq/releases/download/v$YQ_VERSION/yq_linux_amd64 -O /tools/yq && \
+        chmod 0755 /tools/yq ; \
+    elif [ "$TARGETARCH" = "arm64" ]; then \
+        wget -nv https://github.com/mikefarah/yq/releases/download/v$YQ_VERSION/yq_linux_arm64 -O /tools/yq && \
+        chmod 0755 /tools/yq ; \
+    else \
+        echo "Unsupported TARGETARCH: $TARGETARCH" && exit 1 ; \
```

Comment:
There’s no plan (for now and the foreseeable future) to ever try building the ODK on anything else than x86-64 and arm64, but I like the carefulness. :)